test/mockupdb/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 MongoDB, Inc.
```

Comment:
You'll notice that the test suite is failing because `pytohn setup.py test` attempts to run tests in this directory. We can fix that problem by removing this init file.